patroni/ctl.py

```diff
@@ -241,24 +242,28 @@ def watching(w, watch, max_count=None, clear=True):
         yield 0
 
 
-def get_all_members(obj, cluster, group, role='master'):
+def get_all_members(obj, cluster, group, role='leader'):
     clusters = {0: cluster}
     if obj.get('citus') and group is None:
         clusters.update(cluster.workers)
-    if role == 'master':
+    if role in ('leader', 'master', 'primary', 'standby-leader'):
+        role = {'primary': 'master', 'standby-leader': 'standby_leader'}.get(role, role)
```

Comment:
Looks like `master` is still the actual value, so still a "first class citizen", rather than replaced with `primary`. Shouldn't this be replaced both here and where it is being retrieved from?

Reply:
Yes, in this place we can reverse primary<->master in dict. 
But, REST API will still return role=master and so will be stored in the member key in DCS.
We can't simply change everything in one shot because it will break backward compatibility.
For example, during leader race all versions up to 2.1.5 are checking for role==master: https://github.com/zalando/patroni/blob/v2.1.5/patroni/ha.py#L42

Reply:
Perhaps there could be data migration in the dcs code that renames stored values? That could be optional in backwards compatible versions and enforced in non-backwards compat versions.

Reply:
There is no data migration required.
Every node maintains its own member key.
The only problem is that the client tooling and new Patroni are required to understand new values.